main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_info["race"]["name"],
+            description=player_info["race"]["description"]
+        )
+
+        for skill in player_info["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+
+        guild, _ = Guild.objects.get_or_create(
+            name=player_info["guild"]["name"],
+            description=player_info["guild"]["description"]
+        ) if player_info["guild"] else (None, False)
+
+        Player.objects.create(
+            nickname=player_name,
+            email=player_info["email"],
+            bio=player_info["bio"],
+            race=race,
+            guild=guild
+        )
```

Comment:
Consider adding error handling for the `Player.objects.create` call. If there are any validation errors (e.g., unique constraints), the script will raise an exception and terminate. You might want to wrap this in a try-except block to handle such cases gracefully.